dbms/src/Storages/Page/V3/CheckpointFile/Proto/common.proto

```diff
@@ -0,0 +1,30 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.#pragma once
+
+syntax = "proto3";
+
+package DB.PS.V3.CheckpointProto;
+
+message WriterInfo {
+    uint64 store_id = 1;
+    string version = 2;
+    string version_git = 3;
```

Comment:
why we need this version_git?

Reply:
We can fill the git commit of the version. It may be helpful for us to troubleshoot issues by precisely pinpoint the exact code commit when we have this file.